plugins/azure/appservice/identityEnabled.js

```diff
@@ -48,5 +54,50 @@ module.exports = {
             // Global checking goes here
             callback(null, results, source);
         });
+    },
+    remediate: function(config, cache, settings, resource, callback) {
+        var remediation_file = settings.remediation_file;
+        var putCall = this.actions.remediate;
+
+        // inputs specific to the plugin
+        var pluginName = 'identityEnabled';
+        var baseUrl = 'https://management.azure.com/{resource}?api-version=2019-08-01';
+        var method = 'PATCH';
+
+        // for logging purposes
+        var webAppNameArr = resource.split('/');
+        var webAppName = webAppNameArr[webAppNameArr.length - 1];
+
+        // create the params necessary for the remediation
+        if (settings.region) {
+            var body = {
+                'location': settings.region,
+                'identity': {
+                    'type': 'SystemAssigned'
```

Comment:
What if user wants to give user assigned managed identity here? can we take that as input and if no input we can fall back to system assigned one.